src/tests/tests.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@nestjs/common';
+import { spawn } from 'child_process';
+import { Readable } from 'stream';
+
+@Injectable()
+export class TestsService {
+  runTests(): Readable {
+    const process = spawn('python', ['path/to/your/script.py']);
```

Comment:
Remove the dummy path.
Provide the path to your script through an environment variable. Or the process.cwd() API.

Reply:
Thanks